app/assets/javascripts/articles.js

```diff
@@ -1,13 +1,21 @@
 App.articleQueue.onIncoming = function(data) {
-  var alertDiv = $('#floating-alert');
+  var alertDiv = document.querySelectorAll('#floating-alert');
```

Comment:
Since only one element on the page can have the id of `#floating-alert`, does it make more sense to use `getElementById()` instead of `querySelectorAll()`?

https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById

Reply:
Or maybe just `querySelector()`

Reply:
True! I committed a change to querySelector() https://github.com/wbholland/website/commit/40b5707597c4db1d3930a620715bda37718d9b0b